test/utilities/miscellaneous.ts

```diff
@@ -103,3 +105,48 @@ export async function getTextEditor(workbench: Workbench, fileName: string): Pro
   const textEditor = (await editorView.openEditor(fileName)) as TextEditor;
   return textEditor;
 }
+
+export async function createCommand(
+  type: string,
+  name: string,
+  folder: string,
+  extension: string
+): Promise<string | undefined> {
+  const workbench = await (await browser.getWorkbench()).wait();
+  await runCommandFromCommandPrompt(workbench, 'View: Clear Output', 1);
+  const inputBox = await runCommandFromCommandPrompt(workbench, `SFDX: Create ${type}`, 1);
+
+  // Set the name of the new component to name.
+  await inputBox.setText(name);
+  await inputBox.confirm();
+  await pause(1);
+
+  // Select the default directory (press Enter/Return).
+  await inputBox.confirm();
+
+  const successNotificationWasFound = await notificationIsPresentWithTimeout(
+    workbench,
+    `SFDX: Create ${type} successfully ran`,
+    TEN_MINUTES
```

Comment:
@CristiCanizales I know it is necessary to wait for a notification to appear, but why such a large wait time?

Reply:
It's the default time we've been using for checking notifications across all the tests. I created [W-15749369](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001rODKCYA4/view) so we can work on improving all of this long times issues and hard pauses we have everywhere.